health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -86,24 +177,7 @@ const CampaignHome = () => {
-                {CampaignRequirementsData?.[0]?.descriptions?.map((item, index) => (
-                  <li key={index} style={{ marginBottom: "0.75rem" }}>
-                    <strong>{t(item.head)}</strong> {t(item.text)}
-                  </li>
-                ))}
-              </ul>
-
-              {/* Footer */}
-              <div style={{ marginTop: "1.5rem" }}>{t(CampaignRequirementsData?.[0]?.footer)}</div>
-            </div>
-          }
+          <AppHelpContent config={AppHelpConfig} groupTextBlocks={true}></AppHelpContent>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)